app/keygen/cmd/decrypt.go

```diff
@@ -11,72 +11,16 @@
 package main
 
 import (
-	"encoding/base64"
 	"encoding/json"
 	"fmt"
 	"log"
 	"os"
-	"strings"
 
 	"github.com/vmware-tanzu/secrets-manager/core/crypto"
-	entity "github.com/vmware-tanzu/secrets-manager/core/entity/reqres/safe/v1"
+	entity "github.com/vmware-tanzu/secrets-manager/core/entity/v1/reqres/safe"
```

Comment:
moving the version higher, so that it will be easier to copy/paste the entire folder hierarchy when we decide to use v2.